scripts/releasing.gradle

```diff
@@ -0,0 +1,61 @@
+apply plugin: 'maven-publish'
+
+task sourceJar(type: Jar) {
+  from sourceSets.main.allJava
+  classifier 'sources'
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+  from javadoc.destinationDir
+  classifier = 'javadoc'
+}
+
+// this is the only way for us to create a pom using the gradle tools
+// usually this same config is used to deploy, but we're deploying
+// using another tool and really just want the pom.
+publishing {
+  publications {
+    release(MavenPublication) {
+      pom.withXml {
+        def root = asNode()
+        root.appendNode('name', 'App Engine Gradle Plugin')
```

Comment:
I thought if we want to e.g. add a new developer, instead of changing code, just change an xml file. Maybe this is not that critical, though.